swan-lake/learn/keeping-ballerina-up-to-date.md

```diff
@@ -174,76 +165,69 @@ Distributions available remotely:
   [1.2.2] jballerina version 1.2.2
   [1.2.3] jballerina version 1.2.3
   [1.2.4] jballerina version 1.2.4
+  [1.2.5] jballerina version 1.2.5
+  [1.2.6] jballerina version 1.2.6
+  [1.2.7] jballerina version 1.2.7
+  [1.2.8] jballerina version 1.2.8
 
 Swan Lake channel
 
-  [slp1b1] Preview 1
+  [slp1] Preview 1
+  [slp2] Preview 2
+  [slp3] Preview 3
+  [slp4] Preview 4
 
 Use 'ballerina help dist' for more information on specific commands.
 
 ```
 
-The star (*) indicates the active distribution.
+> **Note:** The star (*) indicates the active distribution.
 
-### Remove Distributions
+### Removing a Distribution
 
 The `ballerina dist remove <distribution>` command allows you to delete a particular distribution from your local environment. If you’ve been updating Ballerina regularly, you may have accumulated many unused distribution versions. This command helps you to clean them up.
 
 ```sh
-→ ballerina dist remove 1.0.5
-Distribution '1.0.5' successfully removed
+→ ballerina dist remove slp1
```

Comment:
Better not to update documentation with preview release related content.

Reply:
@anupama-pathirage  Any better suggestion? How should I refer to Swan Lake GA, as "slp"?

Reply:
Since this is a common command IMO it is fine to keep the version as 1.0.5 or any other stable version than the swan lake preview release

Reply:
Fixed.